src/components/casestudyandcoretools/Coretools.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
```

Comment:
change this filename to match the name of the component such as Casetools to CaseTools

Reply:
Done